code/modules/reagents/chemistry/holder.dm

```diff
@@ -673,6 +673,9 @@
 			return R.volume
 	return FALSE
 
+/datum/reagents/proc/is_empty()
+	return total_volume <= 0
+
```

Comment:
I dont really see much use for this helper. 
surely it would just be better to do `!total_volume` anyway? i dont think total volume can be negative

Reply:
removed it